action.yml

```
@@ -6,8 +6,16 @@ branding:
   icon: 'settings'
   color: 'gray-dark'
 
+inputs:
+  directories:
+    description: 'Comma-separated list of directories to look for coverage information (e.g. `src,examples`)'
+    required: false
+    default: 'src'
+
 runs:
   using: 'composite'
   steps:
     - run: julia --color=yes "$GITHUB_ACTION_PATH"/main.jl
       shell: bash
+      env:
+        - INPUT_DIRECTORIES
```

ah, according to https://github.com/actions/runner/issues/665#issuecomment-676581170, we need to pass it manually like
```suggestion
      env:
        - INPUT_DIRECTORIES: ${{ inputs.directories }}
```